platform/darwin/src/MGLLocationManager.m

```diff
@@ -0,0 +1,10 @@
+#import "MGLLocationManager_Private.h"
+
+@implementation CLLocationManager (MGLAdditions)
+
+- (CLAuthorizationStatus)authorizationStatus
```

Comment:
I just realized this could be problematic because category methods should be prefixed with the class prefix. What if CLLocationManager defines a class method by this name in the future?